src/test/java/racinggame/RaceTest.java

```diff
@@ -0,0 +1,40 @@
+package racinggame;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+public class RaceTest {
+
+    private static final int INITIAL_CAR_POSITION = new Car().position();
+
+    @Nested
+    @DisplayName("주어진 횟수만큼 자동차의 위치를 전진시키거나 멈춘다.")
+    class Racing {
+
+        @DisplayName("랜덤 값이 4 미만일 때 자동차는 움직이지 않는다.")
+        @ParameterizedTest
+        @ValueSource(ints = {3})
+        void notMove(int rounds) {
+            Race race = RaceFixture.createRaceWithFixedRandomNumber(3);
+            race.run(rounds);
```

Comment:
```suggestion
        @Test
        void notMove() {
            int fixedNumber = 3
            Race race = new Race(cars, new FakeRandomGenerator(fixedNumber));
            race.run(3);
```
- ParamterizedTest를 사용하는 이점을 잘 모르겠어요. 여러 rounds에도 대응된다면, 이를 분명히 드러내는게 좋겠다는 생각이 드네요.
- Fixture Method 이름이 도메인에서 사용하는 용어가 아니다보니 직관적이지 않아서요. Fake 객체를 주입하여 특정 값을 주입한다는걸 명시적으로 드러내는게 더 좋아보이네요.

Reply:
전진 조건을 만족하지 않는 상황인 Race 객체, 전진 조건을 만족하는 상황인 Race 객체를 편하게 생성하기 위해 RaceFixture를 만들어보았는데, 리뷰어님 말씀대로 유지보수 관리 포인트만 늘어나고 객체를 직접 생성하는 것도 그닥 어렵지 않아보여 제거하였습니다.